docs/user-guide/getting-started/install.md

```diff
@@ -5,6 +5,15 @@ Installation
 Step 1: prerequisites
 ---------------------
 
+<details>
+  <summary>Linux or MacOS</summary>
+  Windows users should use [WSL][wsl].
+
+  % TODO: more information?
```

Comment:
Do we really need to TODO here? 
I mean I understand that we need to add more info, but maybe users don't need to see it.

Reply:
they won't, it's a comment